src/reset.css

```diff
@@ -0,0 +1,13 @@
+* {
```

Comment:
I think this file is not a "reset" exactly but something like "base" maybe. But ok :)